### PR TITLE
Fixes being unable to spraypaint

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

`.atom_colours[atom_colours]` seems like a copy paste error.

I assume it was either intending to check for same color, or check washable color for same color. So I put in both

## Changelog

:cl: Melbert
fix: Fixed being unable to spraypaint things
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
how the hell did i not catch that balloon alert

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -824,8 +824,8 @@
 		var/fraction = min(1, . / reagents.maximum_volume)
 		reagents.expose(carbon_target, VAPOR, fraction * volume_multiplier)
 
-	else if(actually_paints && target.atom_colours[atom_colours] == paint_color)
-		balloon_alert(user, "[target.p_theyre()] already of that color.")
+	else if(actually_paints && (target.color == paint_color || target.atom_colours?[WASHABLE_COLOUR_PRIORITY] == paint_color))
```
Comment (LemonInTheDark) [inline code/game/objects/items/crayons.dm:827]: we should have a getter on atom_colours for the highest priority below some threshold color
this seems a bit odd

**Comment by Ghommie** [comment]
how the hell did i not catch that issue while testing it

File: code/game/atoms.dm [hunk 2]
```diff
@@ -1186,6 +1186,23 @@
...
+	// make sure uppertext hex strings don't mess with lowertext hex strings
+	looking_for_color = lowertext(looking_for_color)
+
+	if(!LAZYLEN(atom_colours))
+		// no atom colors list has been set up, just check the color var
+		return lowertext(color) == looking_for_color
+
+	for(var/i in min_priority_index to max_priority_index)
+		if(lowertext(atom_colours[i]) == looking_for_color)
+			return TRUE
```
Comment (LemonInTheDark) [inline code/game/atoms.dm:1203]: I'm not really sure this is what we want here on reflecting on it. If washable isn't set, don't we want to set it, regardless of what else is setting color?

Comment (LemonInTheDark) [reply code/game/atoms.dm:1203]: basically I dumb

**Pull request merged**